preview-src/index.css

```diff
@@ -142,6 +142,16 @@ button:hover:enabled, button:focus:enabled {
 	cursor: pointer;
 }
 
+body button.checkedOut{
+	opacity: 1 !important;
+	background-color: transparent;
+}
+
+body button svg{

```

Comment:
nitpicky comment: could you add a space before the { here and above? 😄

Reply:
@RMacfarlane you got it, updated the rest of the css to make sure there's a space before the `{`